### PR TITLE
Corail Woodcutter 翻译提交

### PR DESCRIPTION
**Pull request by sea-turtle0**
- [x] 我已**仔细**阅读贡献指南 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)；
- [x] 我已确认英文原文（如 en_us.json）存在且完整，内容与中文对应；
- [x] 我已阅读并同意按 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh) 协议发布我的作品；
- [ ] 刷新 PR 的标签/状态，有需要再点击；

### CODE REVIEW

**Comment by sea-turtle0** [comment]
/mv "projects/corail_woodcutter/assets/corail_woodcutter/" "projects/corail_woodcutter/assets/corail-woodcutter/"

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail_woodcutter/" "projects/1.18/assets/corail-woodcutter/"

**Comment by mamaruo** [comment]
我对比了“库内1.16”和“本PR”的翻译，结果如下：

### 两者皆有且中文有差异

| **键** | **英文** | **库内1.16** | **本PR** |
|---|---|---|---|
| block.corail_woodcutter.tropicraft_mahogany_woodcutter | Mahogany Sawmill | 桃心木锯木机 | 桃花心木锯木机 |
| item.corail_woodcutter.woodcutter.desc | A block for wood craftsmanship being similar to the stonecutter | 与切石机类似，可以处理各种木头 | 一种类似于切石机的木制工作方块 |
| item.corail_woodcutter.woodcutter.inventory | Right-click by sneaking to open it in inventory | 潜行右击可在手中打开锯木机 | 按住Shift+鼠标右键以打开工作界面 |
| corail_woodcutter.config.general.open_woodcutter_in_inventory | Allows to open the woodcutter in inventory with right-click while sneaking | 允许潜行右击打开锯木机 | 可以按住Shift时鼠标右键打开物品栏中的锯木机 |
| stat.corail_woodcutter.interact_with_sawmill | Interactions with Sawmill | 与锯木机互动次数 | 与锯木厂交互 |

### 只在 库内1.16 中有

| **键** | **英文** | **库内1.16** | **本PR** |
|---|---|---|---|
| block.corail_woodcutter.black_stained_woodcutter | 🚫 | 黑色染色木锯木机 | 🚫 |
| block.corail_woodcutter.blue_stained_woodcutter | 🚫 | 蓝色染色木锯木机 | 🚫 |
| block.corail_woodcutter.brown_stained_woodcutter | 🚫 | 棕色染色木锯木机 | 🚫 |
| block.corail_woodcutter.cyan_stained_woodcutter | 🚫 | 青色染色木锯木机 | 🚫 |
| block.corail_woodcutter.gray_stained_woodcutter | 🚫 | 灰色染色木锯木机 | 🚫 |
| block.corail_woodcutter.green_stained_woodcutter | 🚫 | 绿色染色木锯木机 | 🚫 |
| block.corail_woodcutter.light_blue_stained_woodcutter | 🚫 | 淡蓝色染色木锯木机 | 🚫 |
| block.corail_woodcutter.light_gray_stained_woodcutter | 🚫 | 淡灰色染色木锯木机 | 🚫 |
| block.corail_woodcutter.lime_stained_woodcutter | 🚫 | 黄绿色染色木锯木机 | 🚫 |
| block.corail_woodcutter.magenta_stained_woodcutter | 🚫 | 品红色染色木锯木机 | 🚫 |
| block.corail_woodcutter.orange_stained_woodcutter | 🚫 | 橙色染色木锯木机 | 🚫 |
| block.corail_woodcutter.pink_stained_woodcutter | 🚫 | 粉红色染色木锯木机 | 🚫 |
| block.corail_woodcutter.purple_stained_woodcutter | 🚫 | 紫色染色木锯木机 | 🚫 |
| block.corail_woodcutter.red_stained_woodcutter | 🚫 | 红色染色木锯木机 | 🚫 |
| block.corail_woodcutter.white_stained_woodcutter | 🚫 | 白色染色木锯木机 | 🚫 |
| block.corail_woodcutter.yellow_stained_woodcutter | 🚫 | 黄色染色木锯木机 | 🚫 |
| block.corail_woodcutter.trans_woodcutter | 🚫 | 变化木锯木机 | 🚫 |
| block.corail_woodcutter.mine_woodcutter | 🚫 | 矿石木锯木机 | 🚫 |
| block.corail_woodcutter.sort_woodcutter | 🚫 | 分类木锯木机 | 🚫 |

### 只在 本PR 中有

| **键** | **英文** | **库内1.16** | **本PR** |
|---|---|---|---|
| block.corail_woodcutter.transformation_woodcutter | Transwood Sawmill | 🚫 | 变化木锯木机 |
| block.corail_woodcutter.mining_woodcutter | Minewood Sawmill | 🚫 | 矿石木锯木机 |
| block.corail_woodcutter.sorting_woodcutter | Sortwood Sawmill | 🚫 | 分类木锯木机 |
| corail_woodcutter.command.usage | Generate the wooden crafting recipes of a mod in the Corail Woodcutter format in your config folder
 /woodcutter info
 /woodcutter datapack [generate|apply|remove] <modid>
apply & remove are useable in singleplayer and the datapack must be generated before applying it | 🚫 | 在配置文件夹中以Corail Woodcutter格式生成mod的锯木机配方
 /woodcutter info
 /woodcutter datapack [generate|apply|remove] <modid>
在单人游戏中可以使用或删除，在应用数据包之前必须先生成数据包 |
| corail_woodcutter.message.datapack.generate.success | The datapack was successfully generated in your config folder (%d recipes) | 🚫 | 数据包已在你的配置文件夹中成功生成（%d 个配方） |
| corail_woodcutter.message.datapack.generate.fail | The datapack wasn't generated: %s | 🚫 | 未生成数据包：%s |
| corail_woodcutter.message.file.delete.fail | File can't be deleted: %s | 🚫 | 无法删除文件：%s |
| corail_woodcutter.message.file.write.fail | File can't be written: %s | 🚫 | 无法写入文件：%s |
| corail_woodcutter.message.file.copy.fail | File can't be copied: %s | 🚫 | 无法复制文件：%s |
| corail_woodcutter.message.folder.create.fail | Folder can't be created: %s | 🚫 | 无法创建文件夹：%s |
| corail_woodcutter.message.zip.create.fail | Zip creation failed | 🚫 | Zip创建失败 |
| corail_woodcutter.message.mcmeta.create.fail | The pack.mcmeta file could not be written | 🚫 | 无法写入pack.mcmeta文件 |
| corail_woodcutter.message.no_valid_recipe_for_modid | No valid recipes were found for the modid: %s | 🚫 | 找不到修改后的有效配方：%s |
| corail_woodcutter.message.invalid_modid | Invalid modid | 🚫 | 修改无效 |
| corail_woodcutter.message.datapack.not_generated | No generated datapack for modid: %s | 🚫 | 没有为modid生成数据包：%s |
| corail_woodcutter.message.datapack.apply.success | The datapack was added | 🚫 | 已添加数据包 |
| corail_woodcutter.message.datapack.apply.fail | The datapack wasn't applied: %s | 🚫 | 未应用数据包：%s |
| corail_woodcutter.message.datapack.remove.success | The datapack was removed | 🚫 | 数据包已删除 |
| corail_woodcutter.message.datapack.remove.absent | The datapack was absent | 🚫 | 数据包不存在 |
| corail_woodcutter.message.datapack.remove.fail | The datapack wasn't not removed: %s | 🚫 | 未删除数据包：%s |

**Comment by sea-turtle0** [comment]
> ## 机器翻译警告！
> image
> 
> 重申（[原文见此](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md#:~:text=%E6%9C%AC%E9%A1%B9%E7%9B%AE%E5%AF%B9%E6%9C%BA%E7%BF%BB%E3%80%81%E7%94%9F%E7%A1%AC%E7%BF%BB%E8%AF%91%E5%B9%B6%E4%B8%8D%E5%8F%8B%E5%A5%BD%EF%BC%8C%E8%BF%98%E8%AF%B7%E5%9C%A8%E6%8F%90%E4%BA%A4%E5%85%B7%E6%9C%89%E8%BF%99%E4%BA%9B%E7%89%B9%E5%BE%81%E7%9A%84%E6%96%87%E4%BB%B6%E6%97%B6%E6%B7%B1%E6%80%9D%E7%86%9F%E8%99%91%E3%80%82)）：
> 
> ### 本项目对机翻、生硬翻译并不友好，还请在提交具有这些特征的文件时深思熟虑。
> 我对比了“库内 1.16”和“本 PR”的翻译，结果如下：
> 
> ### 两者皆有且中文有差异
> **键**	**英文**	**库内 1.16**	**本 PR**
> block.corail_woodcutter.tropicraft_mahogany_woodcutter	Mahogany Sawmill	桃心木锯木机	桃花心木锯木机
> item.corail_woodcutter.woodcutter.desc	A block for wood craftsmanship being similar to the stonecutter	与切石机类似，可以处理各种木头	一种类似于切石机的木制工作方块
> item.corail_woodcutter.woodcutter.inventory	Right-click by sneaking to open it in inventory	潜行右击可在手中打开锯木机	按住Shift+鼠标右键以打开工作界面
> corail_woodcutter.config.general.open_woodcutter_in_inventory	Allows to open the woodcutter in inventory with right-click while sneaking	允许潜行右击打开锯木机	可以按住Shift时鼠标右键打开物品栏中的锯木机
> stat.corail_woodcutter.interact_with_sawmill	Interactions with Sawmill	与锯木机互动次数	与锯木厂交互
> ### 只在 库内 1.16 中有
> **键**	**英文**	**库内 1.16**	**本 PR**
> block.corail_woodcutter.black_stained_woodcutter	🚫	黑色染色木锯木机	🚫
> block.corail_woodcutter.blue_stained_woodcutter	🚫	蓝色染色木锯木机	🚫
> block.corail_woodcutter.brown_stained_woodcutter	🚫	棕色染色木锯木机	🚫
> block.corail_woodcutter.cyan_stained_woodcutter	🚫	青色染色木锯木机	🚫
> block.corail_woodcutter.gray_stained_woodcutter	🚫	灰色染色木锯木机	🚫
> block.corail_woodcutter.green_stained_woodcutter	🚫	绿色染色木锯木机	🚫
> block.corail_woodcutter.light_blue_stained_woodcutter	🚫	淡蓝色染色木锯木机	🚫
> block.corail_woodcutter.light_gray_stained_woodcutter	🚫	淡灰色染色木锯木机	🚫
> block.corail_woodcutter.lime_stained_woodcutter	🚫	黄绿色染色木锯木机	🚫
> block.corail_woodcutter.magenta_stained_woodcutter	🚫	品红色染色木锯木机	🚫
> block.corail_woodcutter.orange_stained_woodcutter	🚫	橙色染色木锯木机	🚫
> block.corail_woodcutter.pink_stained_woodcutter	🚫	粉红色染色木锯木机	🚫
> block.corail_woodcutter.purple_stained_woodcutter	🚫	紫色染色木锯木机	🚫
> block.corail_woodcutter.red_stained_woodcutter	🚫	红色染色木锯木机	🚫
> block.corail_woodcutter.white_stained_woodcutter	🚫	白色染色木锯木机	🚫
> block.corail_woodcutter.yellow_stained_woodcutter	🚫	黄色染色木锯木机	🚫
> block.corail_woodcutter.trans_woodcutter	🚫	变化木锯木机	🚫
> block.corail_woodcutter.mine_woodcutter	🚫	矿石木锯木机	🚫
> block.corail_woodcutter.sort_woodcutter	🚫	分类木锯木机	🚫
> ### 只在 本 PR 中有
> **键**	**英文**	**库内 1.16**	**本 PR**
> block.corail_woodcutter.transformation_woodcutter	Transwood Sawmill	🚫	变化木锯木机
> block.corail_woodcutter.mining_woodcutter	Minewood Sawmill	🚫	矿石木锯木机
> block.corail_woodcutter.sorting_woodcutter	Sortwood Sawmill	🚫	分类木锯木机
> corail_woodcutter.command.usage	Generate the wooden crafting recipes of a mod in the Corail Woodcutter format in your config folder\n /woodcutter info\n /woodcutter datapack [generate|apply|remove] \napply & remove are useable in singleplayer and the datapack must be generated before applying it	🚫	在配置文件夹中以Corail Woodcutter格式生成mod的锯木机配方\n /woodcutter info\n /woodcutter datapack [generate|apply|remove] \n在单人游戏中可以使用或删除，在应用数据包之前必须先生成数据包
> corail_woodcutter.message.datapack.generate.success	The datapack was successfully generated in your config folder (%d recipes)	🚫	数据包已在你的配置文件夹中成功生成（%d 个配方）
> corail_woodcutter.message.datapack.generate.fail	The datapack wasn't generated: %s	🚫	未生成数据包：%s
> corail_woodcutter.message.file.delete.fail	File can't be deleted: %s	🚫	无法删除文件：%s
> corail_woodcutter.message.file.write.fail	File can't be written: %s	🚫	无法写入文件：%s
> corail_woodcutter.message.file.copy.fail	File can't be copied: %s	🚫	无法复制文件：%s
> corail_woodcutter.message.folder.create.fail	Folder can't be created: %s	🚫	无法创建文件夹：%s
> corail_woodcutter.message.zip.create.fail	Zip creation failed	🚫	Zip创建失败
> corail_woodcutter.message.mcmeta.create.fail	The pack.mcmeta file could not be written	🚫	无法写入pack.mcmeta文件
> corail_woodcutter.message.no_valid_recipe_for_modid	No valid recipes were found for the modid: %s	🚫	找不到修改后的有效配方：%s
> corail_woodcutter.message.invalid_modid	Invalid modid	🚫	修改无效
> corail_woodcutter.message.datapack.not_generated	No generated datapack for modid: %s	🚫	没有为modid生成数据包：%s
> corail_woodcutter.message.datapack.apply.success	The datapack was added	🚫	已添加数据包
> corail_woodcutter.message.datapack.apply.fail	The datapack wasn't applied: %s	🚫	未应用数据包：%s
> corail_woodcutter.message.datapack.remove.success	The datapack was removed	🚫	数据包已删除
> corail_woodcutter.message.datapack.remove.absent	The datapack was absent	🚫	数据包不存在
> corail_woodcutter.message.datapack.remove.fail	The datapack wasn't not removed: %s	🚫	未删除数据包：%s

修改完毕

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "block.corail_woodcutter.oak_woodcutter": "橡木锯木机",
+  "block.corail_woodcutter.spruce_woodcutter": "云杉木锯木机",
+  "block.corail_woodcutter.birch_woodcutter": "白桦木锯木机",
+  "block.corail_woodcutter.jungle_woodcutter": "丛林木锯木机",
+  "block.corail_woodcutter.acacia_woodcutter": "金合欢木锯木机",
+  "block.corail_woodcutter.dark_oak_woodcutter": "深色橡木锯木机",
+  "block.corail_woodcutter.crimson_woodcutter": "绯红木锯木机",
+  "block.corail_woodcutter.warped_woodcutter": "诡异木锯木机",
+  "block.corail_woodcutter.cherry_woodcutter": "樱桃木锯木机",
+  "block.corail_woodcutter.dead_woodcutter": "垂死木锯木机",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:11]:
```suggestion
  "block.corail_woodcutter.dead_woodcutter": "枯死木锯木机",
```
BOP

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  "block.corail_woodcutter.fir_woodcutter": "杉木锯木机",
+  "block.corail_woodcutter.hellbark_woodcutter": "地狱皮木锯木机",
+  "block.corail_woodcutter.jacaranda_woodcutter": "蓝花楹木锯木机",
+  "block.corail_woodcutter.magic_woodcutter": "魔法木锯木机",
+  "block.corail_woodcutter.mahogany_woodcutter": "桃心木锯木机",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:16]:
```suggestion
  "block.corail_woodcutter.mahogany_woodcutter": "桃花心木锯木机",
```
库内原本的要改了

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  "block.corail_woodcutter.bogshroom_woodcutter": "藓沼菇锯木机",
+  "block.corail_woodcutter.dark_willow_woodcutter": "晦暗柳锯木机",
+  "block.corail_woodcutter.dead_wood_woodcutter": "枯萎木锯木机",
+  "block.corail_woodcutter.dewshroom_woodcutter": "露水菇锯木机",
+  "block.corail_woodcutter.nightshroom_woodcutter": "暗夜菇锯木机",
+  "block.corail_woodcutter.shadowroot_woodcutter": "暗影根锯木机",
+  "block.corail_woodcutter.viridshroom_woodcutter": "翠碧菇锯木机",
+  "block.corail_woodcutter.twilight_oak_woodcutter": "暮色橡木锯木机",
+  "block.corail_woodcutter.canopy_woodcutter": "苍穹木锯木机",
+  "block.corail_woodcutter.mangrove_woodcutter": "红木锯木机",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:30]:
```suggestion
  "block.corail_woodcutter.mangrove_woodcutter": "红树木锯木机",
```

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  "block.corail_woodcutter.dark_woodcutter": "黑木锯木机",
+  "block.corail_woodcutter.time_woodcutter": "时光木锯木机",
+  "block.corail_woodcutter.transformation_woodcutter": "变化木锯木机",
+  "block.corail_woodcutter.mining_woodcutter": "矿石木锯木机",
+  "block.corail_woodcutter.sorting_woodcutter": "分类木锯木机",
+  "block.corail_woodcutter.tropicraft_palm_woodcutter": "棕榈木锯木机",
+  "block.corail_woodcutter.tropicraft_mahogany_woodcutter": "桃心木锯木机",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:37]:
```suggestion
  "block.corail_woodcutter.tropicraft_mahogany_woodcutter": "桃花心木锯木机",
```

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  "container.corail_woodcutter.woodcutter": "锯木机",
+  "item.corail_woodcutter.woodcutter.desc": "与切石机类似，可以处理各种木头",
+  "item.corail_woodcutter.woodcutter.inventory": "潜行右击可在手中打开锯木机",
+  "corail_woodcutter.config.general.open_woodcutter_in_inventory": "允许潜行右击打开锯木机",
+  "stat.corail_woodcutter.interact_with_sawmill": "与锯木机互动次数",
+  "corail_woodcutter.command.usage": "在配置文件夹中以Corail Woodcutter格式生成mod的锯木机配方\n /woodcutter info\n /woodcutter datapack [generate|apply|remove] <modid>\n在单人游戏中可以使用或删除，在应用数据包之前必须先生成数据包",
+  "corail_woodcutter.message.datapack.generate.success": "数据包已在你的配置文件夹中成功生成（%d 个配方）",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:44]:
```suggestion
  "corail_woodcutter.message.datapack.generate.success": "数据包已在你的配置文件夹中成功生成（%d个配方）",
```

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  "block.corail_woodcutter.mining_woodcutter": "矿石木锯木机",
+  "block.corail_woodcutter.sorting_woodcutter": "分类木锯木机",
+  "block.corail_woodcutter.tropicraft_palm_woodcutter": "棕榈木锯木机",
+  "block.corail_woodcutter.tropicraft_mahogany_woodcutter": "桃心木锯木机",
+  "container.corail_woodcutter.woodcutter": "锯木机",
+  "item.corail_woodcutter.woodcutter.desc": "与切石机类似，可以处理各种木头",
+  "item.corail_woodcutter.woodcutter.inventory": "潜行右击可在手中打开锯木机",
+  "corail_woodcutter.config.general.open_woodcutter_in_inventory": "允许潜行右击打开锯木机",
+  "stat.corail_woodcutter.interact_with_sawmill": "与锯木机互动次数",
+  "corail_woodcutter.command.usage": "在配置文件夹中以Corail Woodcutter格式生成mod的锯木机配方\n /woodcutter info\n /woodcutter datapack [generate|apply|remove] <modid>\n在单人游戏中可以使用或删除，在应用数据包之前必须先生成数据包",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:43]:
```suggestion
  "corail_woodcutter.command.usage": "在配置文件夹中以Corail Woodcutter格式生成mod的锯木机配方\n /woodcutter info\n /woodcutter datapack [generate|apply|remove] <modid>\napply和remove仅在单人游戏可用，且应用数据包之前必须先生成数据包",
```

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  "corail_woodcutter.message.datapack.generate.success": "数据包已在你的配置文件夹中成功生成（%d 个配方）",
+  "corail_woodcutter.message.datapack.generate.fail": "未生成数据包：%s",
+  "corail_woodcutter.message.file.delete.fail": "无法删除文件：%s",
+  "corail_woodcutter.message.file.write.fail": "无法写入文件：%s",
+  "corail_woodcutter.message.file.copy.fail": "无法复制文件：%s",
+  "corail_woodcutter.message.folder.create.fail": "无法创建文件夹：%s",
+  "corail_woodcutter.message.zip.create.fail": "Zip创建失败",
```
Comment (ChuijkYahus) [inline projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:50]:
```suggestion
  "corail_woodcutter.message.zip.create.fail": "zip创建失败",
```

**Review by ChuijkYahus — CHANGES_REQUESTED** [review]
请注意更新文件路径
图片
见[CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json (hunk 6 above)
Comment (mamaruo) [reply projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json:43]: 关键字保留不译一例

**Comment by sea-turtle0** [comment]
> 请注意更新文件路径 图片 见[CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)

了解

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json (hunk 1 above)
Comment (sea-turtle0) [reply projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json]: 抱歉，已修改

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json (hunk 2 above)
Comment (sea-turtle0) [reply projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json]: 抱歉，已修改

File: projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json (hunk 6 above)
Comment (sea-turtle0) [reply projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/zh_cn.json]: 了解，抱歉，已修改

**Comment by ChuijkYahus** [comment]
> > 请注意更新文件路径 图片 见[CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)
> 
> 了解

这路径没改好啊，多了一层文件夹

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/" "projects/1.18/assets/corail-woodcutter/corail_woodcutter"

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/corail_woodcutter/lang/" "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/"

**Comment by sea-turtle0** [comment]
啊嘞，看来变成多套一层lang文件夹了（

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/lang/" "projects/1.18/assets/corail-woodcutter/lang/ /"

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/ /lang/" "projects/1.18/assets/corail-woodcutter/lang/ zh_cn.json"

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/ /lang/" "projects/1.18/assets/corail-woodcutter/lang"

**Comment by sea-turtle0** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/ /lang/" "projects/1.18/assets/corail-woodcutter/lang/"

**Comment by ChuijkYahus** [comment]
/mv-recursive "projects/1.18/assets/corail-woodcutter/lang/" "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/"

**Comment by ChuijkYahus** [comment]
/mv "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/ /lang/" "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/"

**Comment by ChuijkYahus** [comment]
/mv-recursive "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/lang/" "projects/1.18/assets/corail-woodcutter/corail_woodcutter/lang/"

**Comment by ChuijkYahus** [comment]
我的建议是，之后直接在github.dev里面改，用命令不方便

**Comment by sea-turtle0** [comment]
> 我的建议是，之后直接在github.dev里面改，用命令不方便

谢谢，我傻了

**Review by ChuijkYahus — APPROVED** [review]

**Pull request merged**